### DISCUSSION TITLE
No 'vasp_std' file

### DISCUSSION THREAD

**Discussion by Ruichen-Li** [ftherrien/VaspGibbs#3] [VaspGibbs Forum]
Hi, I'd like to ask what should I do if I can not find the file 'vasp_std', the version I used is vasp_ncl. 
![image](https://user-images.githubusercontent.com/91064644/175857034-6ee1e8f8-0b0a-4988-b4ad-505e6e4951b1.png)

**Comment by ftherrien** [comment]
Hey Ruichen,

Thanks for your question. You can set the vasp executable using the --vasp option.

`vasp_gibbs --vasp "vasp_ncl"`

Let me know if you have any other questions

**Reply by Ruichen-Li** [reply]
Thanks for your reply, that works :)
